src/components/DpButton/DpButton.vue

```diff
@@ -89,22 +143,55 @@ export default {
       return [
         'btn',
         this.busy && 'is-busy pointer-events-none',
-        ['primary', 'secondary', 'warning'].includes(this.color) && `btn--${this.color}`,
-        ['outline'].includes(this.variant) && `btn--${this.variant}`
+        ['primary', 'secondary', 'warning'].includes(this.color) && classes.color[this.color],
+        ['solid', 'outline', 'subtle'].includes(this.variant) && classes.variant[this.variant]
       ]
     },
 
     element () {
       return this.isButtonElement ? 'button' : 'a'
     },
 
+    /*
+     * Render a tooltip to show when hovering icon-only buttons.
+     */
+    iconTooltip () {
```

Comment:
If I set `iconAfter` and `hideText = true`, the tooltip will not be displayed. We could handle this by adding a condition `|| (iconAfter !== '' && hideText` to the `v-if="icon"` span, and adjusting the condition on the `v-if="iconAfter"` span:
`v-if="iconAfter && !hideText"`

Reply:
good catch! however i think it is enough to just show the tooltip on the iconAfter element: 880ed8149eadb540021fabbb272dd176b9da5696

the tooltip will be rendered twice, if the button is used with two icons but without text. but that is an absurd case anyways.